src/appleseed.studio/mainwindow/rendering/rendertab.h

```diff
@@ -86,6 +87,8 @@ class RenderTab
     CameraController* get_camera_controller() const;
     ScenePickingHandler* get_scene_picking_handler() const;
 
+    void set_material_drop_handler(MaterialDropHandler *material_drop_handler);
```

Comment:
Ownership of the handler is transfered here but it's not clear from the API. Take a `std::unique_ptr<>` instead.